dds/DCPS/transport/multicast/MulticastReceiveStrategy.cpp

```diff
@@ -18,6 +18,9 @@ namespace DCPS {
 MulticastReceiveStrategy::MulticastReceiveStrategy(MulticastDataLink* link)
   : link_(link)
 {
+#if WIN32
```

Comment:
Could this be made unconditional?  ACE Reactor reference counting seems like a good idea in general.

If it remains conditional, see the preprocessor conditions in MulticastInst.cpp:129

Reply:
I don't see why not, I did it because it's a Windows problem. Would you think it would be a good idea to apply it to other Transport Strategies as well, since it seems at least some of them are also `ACE_Event_Handler`s?

Reply:
Yes, I think that makes sense.  It would apply to all the network-based transports (not shmem).

Reply:
Sounds also to me a good thing to always use reference counting